cartographer_ros/cartographer_ros/node_constants.h

```diff
@@ -33,6 +33,8 @@ constexpr char kSubmapQueryServiceName[] = "submap_query";
 constexpr char kStartTrajectoryServiceName[] = "start_trajectory";
 constexpr char kWriteAssetsServiceName[] = "write_assets";
 constexpr char kTrajectoryNodesListTopic[] = "trajectory_nodes_list";
+constexpr char kConstraintsListTopic[] = "constraints_list";
```

Comment:
Use singular? Calling it a list already says it can contain multiple items, i.e. `kConstraintListTopic`, `constraint_list`, etc